spec/features/articles/search_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.feature "Search Article" do
+  let!(:target_articles) do
```

Comment:
I think also you can check cases when
- there is article that unsearchable
- there is not published article

```ruby
let(:article_1) { create(:article, :published, title: "How to add search to Rails project?") }
let(:article_2) { create(:article, :published, content: "Why you not use pg_search?") }
let(:article_3) { create(:article, :published, content: "Invalid") }
let(:article_4) { create(:article, :not_published, content: "Search") } 
```

```ruby
expect(page).to have_content(article_1.title)
expect(page).to have_content(article_2.title)

expect(page).not_to have_content(article_3.title)
expect(page).not_to have_content(article_4.title)
```